common/lib/dependabot/pull_request_creator/message_builder/link_and_mention_sanitizer.rb

```diff
@@ -19,7 +19,7 @@ class LinkAndMentionSanitizer
         EOS_REGEX = /\z/.freeze
         # We rely on GitHub to do the HTML sanitization
         COMMONMARKER_OPTIONS = %i(
-          UNSAFE GITHUB_PRE_LANG FULL_INFO_STRING
+          DEFAULT GITHUB_PRE_LANG FULL_INFO_STRING
```

Comment:
Oh wait will this break all previous sections that have been turned into HTML bu this step?

Reply:
Yeah looks like we re-parse all the already sanitized markdown that's now HTML here https://github.com/dependabot/dependabot-core/blob/master/common/lib/dependabot/pull_request_creator/message_builder.rb#L326 - maybe ok if we get rid of that step? Not sure it's adding anything

Reply:
Ah yes you're right, I think it should be safe to get rid of that step

Reply:
Looks like we sanitize all cascades except for `maintainer_changes_cascade`, might work as expected if we move `sanitize_links_and_mentions` from `metadata_cascades_for_dep` to `maintainer_changes_cascade`

Reply:
I opted to do a final "unsafe" pass of the sanitization, to ensure we don't mention people by accident.

Reply:
Also passed `maintainer_changes_cascade` through the sanitization to be sure, nice spot, ty!